java/arcs/android/crdt/ReferencableProto.kt

```diff
@@ -0,0 +1,36 @@
+package arcs.android.crdt
+
+import android.os.Parcel
+import arcs.android.util.readProto
+import arcs.core.common.Referencable
+import arcs.core.crdt.CrdtEntity
+import arcs.core.data.RawEntity
+import arcs.core.storage.Reference
+
+/** Constructs a [Referencable] from the given [ReferencableProto]. */
+fun fromProto(proto: ReferencableProto): Referencable? = when (proto.referencableCase) {
```

Comment:
And here.. noticing a pattern - I'll just stop here

Reply:
Done